envoy/upstream/upstream.h

```diff
@@ -765,6 +765,7 @@ MAKE_STATS_STRUCT(ClusterLbStats, ClusterLbStatNames, ALL_CLUSTER_LB_STATS);
  */
 MAKE_STAT_NAMES_STRUCT(ClusterTrafficStatNames, ALL_CLUSTER_TRAFFIC_STATS);
 MAKE_STATS_STRUCT(ClusterTrafficStats, ClusterTrafficStatNames, ALL_CLUSTER_TRAFFIC_STATS);
+using ClusterTrafficStatsPtr = std::unique_ptr<ClusterTrafficStats>;
```

Comment:
I think it'd be better for the CL author not to resolve comments unless it was a simple suggestion that was implemented.

Otherwise that causes the reviewer to re-review and re-discover the same issue.

I think if we change the nickname to `LazyClusterTrafficStats' then it can be commented right here and not at call sites, and that's what I'm learning toward.